scripts/ci/install-minikube.sh

```diff
@@ -57,22 +66,26 @@ uninstall() {
 
 stop() {
 	check_minikube
-	sudo $(which minikube) delete || true
+	sudo systemctl stop localkube || true
+	sudo minikube delete || true
 	sudo rm -rf ~/.minikube ~/.kube
 }
 
 start() {
 	check_minikube
-	sudo -E CHANGE_MINIKUBE_NONE_USER=true $(which minikube) \
-		--vm-driver=${MINIKUBE_VM_DRIVER} \
-		${MINIKUBE_FLAGS} start
-	sudo $(which minikube) status
+	sudo CHANGE_MINIKUBE_NONE_USER=true minikube start ${MINIKUBE_FLAGS}
```

Comment:
you don't need to do this hacky clean/move the root profile if you use 
sudo -E (as before) as it will export user HOME environment variable

Reply:
using export now